### ISSUE TITLE
Add automated data updates to epimodel

### ISSUE THREAD

**Issue by gavento** [epidemics/covid#393]
We can easily have:
* Foretold updates
* Johns Hopkins updates

Should be doable with github actions.

Would be nice to have:
* Countermeasure updates (blocked on notion.so downloads and David's manual exports)

**Comment by mathijshenquet** [comment]
I have a branch where I was experimenting with this here: https://github.com/epidemics/epimodel/blob/growth-csv/.github/workflows/tasks.yml

Would you want it to autocommit to data?

Also @wolverdude is working on foretold containment exports for FE and maybe his work can be used to automate your last point.

**Comment by hnykda** [comment]
I would prefer k8s cronjobs, especially if it gets heavier (like downloading an HDF file etc). But no problem trying it via actions first too.

**Comment by hnykda** [comment]
I make a very quick example how I imagine it could work (but it's not functional yet, I think it would take me another hour to finish it): https://github.com/epidemics/epimodel/pull/39

**Comment by hazarane** [comment]
Closing this as the content of the ticket is not relevant anymore. Any possible points for automation and improvement will be implemented separately upon request.
